pkg/scheduler/internal/heap/heap.go

```diff
@@ -256,3 +256,7 @@ func NewHeapWithRecorder(keyFn KeyFunc, lessFn LessFunc, metricRecorder metrics.
 		metricRecorder: metricRecorder,
 	}
 }
+
+// LessFunc is a function that receives two items and returns true if the first
+// item should be placed before the second one when the list is sorted.
+type LessFunc func(item1, item2 interface{}) bool
```

Comment:
you can make this a private alias instead. There's not much benefit on having it as defined type.

Reply:
Chnages as per recommendation.

Reply:
Did you try making it a **private** alias instead?